compiler/src/iree/compiler/Dialect/HAL/Target/ROCM/ROCMTarget.cpp

```diff
@@ -199,9 +202,19 @@ class ROCMTargetBackend final : public TargetBackend {
     iree_hal_rocm_ExecutableDef_start_as_root(builder);
 
     // Link module to Device Library
+    std::string rocmBitcodeDir = clROCMBitcodeDir;
+    if (clROCMBitcodeDir.empty()) {
+      const std::string kPlatformSuffix =  "/_platform_libs/rocm";
+      const std::string kRocmSysPath = "/opt/rocm/amdgcn/bitcode";
```

Comment:
what's the path on windows?

Reply:
Ah you're right need to add the default sys path for Windows! :)

Reply:
I'm going to rework this part...